src/assets/i18n/ja.json

```diff
@@ -1,6 +1,7 @@
 {
   "APP": {
     "LANGUAGE": "言語",
+    "USER_DASHBOARD": "ユーザーダッシュボード」",
```

Comment:
We might have somebody in Sphereon who speaks Spanish and perhaps even Italian, but where did we get the Asian, Scandinavian and Russian translations from?

Reply:
@LDMGN google translate

Reply:
German and english are fine. Please check french with @ChristopheVico. I think that the dutch version should be "Gebruikerspaneel". The ones that we are not sure about should be either removed or replaced with the english version (depending on how Alfresco handles missing values and defaults).

Reply:
@LDMGN removed the uncertain ones and renamed the one for dutch to "Gebruikerspaneel"